webpack/assets/javascripts/react_app/components/PasswordStrength/__tests__/index.test.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { createStore, combineReducers } from 'redux';
+import { Provider } from 'react-redux';
+
+import PasswordStrength, { reducers } from '../index';
+
+const fixtures = {
+  username: { password: 'my-user-name', expected: __('Weak') },
+  email: { password: 'my-user@email.com', expected: __('Weak') },
+  tooShort: { password: '12345', expected: __('Too short') },
+  weak: { password: '123456', expected: __('Weak') },
+  medium: { password: 'qwedsa', expected: __('Medium') },
+  normal: { password: 'QwedsaZx', expected: __('Normal') },
+  strong: { password: 'StrongP@$w0rd', expected: __('Strong') },
+  veryStrong: { password: '|Kkh)Xeu#T8($"P;', expected: __('Very strong') },
+};
+
+document.getElementById = jest.fn(id => ({ value: fixtures[id].password }));
+
+describe('PasswordStrength integration test', () => {
+  const generateStore = () => createStore(combineReducers({ ...reducers }));
```

Comment:
instead of generate store each time, would it be better to add integration setup (helper) ?
something like this: https://github.com/theforeman/foreman/pull/5205/files#diff-6d91af9e34087f473783a4de94e3c806
maybe it better to combined those two PRs?
I took inspiration from [this](https://medium.freecodecamp.org/real-integration-tests-with-react-redux-and-react-router-417125212638) blog

Reply:
Yes, i think it makes sense thanks for pointing out :+1: 

Will update soon